tests/cache/tests.py

```diff
@@ -94,6 +94,11 @@ def test_get_many(self):
         self.assertEqual(cache.get_many(['a', 'c', 'd']), {})
         self.assertEqual(cache.get_many(['a', 'b', 'e']), {})
 
+    def test_get_many_invalid_key(self):
+        msg = "Cache key contains characters that will cause errors if used with memcached: ':1:key with spaces'"
```

Comment:
I'd make the message (with a placeholder for the key) a module constant so it's not repeated.